website/docs/installing-podman/index.md

```diff
@@ -0,0 +1,20 @@
+---
+sidebar_position: 2
+title: Installing Podman
+description: You can install Podman on Windows, macOS, and Linux.
+tags: [podman-desktop, installing, windows, macos, linux]
+keywords: [podman desktop, containers, podman, installing, installation, windows, macos, linux]
+---
+
+Podman Desktop let you choose your container engine, and doesn't install Podman automatically.
+
+Consider installing Podman if you are not required to use another container engine.
+
+On Linux, the Podman container engine runs natively.
+
+On macOS and Windows, running the Podman container engine requires running a Linux distribution on a virtual machine. Podman Desktop can assist you to install the Podman container engine on these platforms.
```

Comment:
This section is a repeat of website/docs/Installation/index.md ? Bit confusing. Because in this part we link to [Installing Podman on Linux] but on the other index, we dont.

Reply:
Nice pick. That's a leftover.

Reply:
I was thinking of breaking down tasks to keep the pull request scope not too large:
 
* In this pull request, I focus mainly on the Installing Podman section. 
* In subsequent pull request(s), refresh and reorganize the Installing Podman Desktop section (probably one for each operating system).